distributed/core.py

```diff
@@ -608,7 +608,7 @@ def coerce_to_rpc(o, **kwargs):
     if isinstance(o, (bytes, str, tuple, list)):
         ip, port = coerce_to_address(o, out=tuple)
         return rpc(ip=ip, port=int(port), **kwargs)
-    elif isinstance(o, IOStream):
+    elif isinstance(o, IOStream) or isinstance(o, SSLIOStream):
```

Comment:
Although actually it looks like `SSLIOStream` inherits from `IOStream`, so this should not be necessary.

```python
In [2]: SSLIOStream.mro()
Out[2]: 
[tornado.iostream.SSLIOStream,
 tornado.iostream.IOStream,
 tornado.iostream.BaseIOStream,
 object]
```